### PR TITLE
Update bokeh plots

### PR DESCRIPTION
**Pull request by mrocklin**
*  Fix size of memory plot
*  Add height of task stream plot (maybe not a good idea)

### CODE REVIEW

**Comment by mrocklin** [comment]
cc @canavandl can you give this four-line PR a quick glance to check if anything seems off to you?

File: distributed/bokeh/components.py [hunk 1]
```diff
@@ -402,6 +402,7 @@ def __init__(self, **kwargs):
 
         mem_plot.add_layout(LinearAxis(), 'left')
         mem_plot.add_layout(LinearAxis(), 'below')
+        mem_plot.yaxis.visible = False
```
Comment (canavandl) [inline distributed/bokeh/components.py]: remove L404 instead

**Comment by canavandl** [comment]
What's the intent of settings the plot_height in the element constructor instead of passing it in as a kwarg? I think doing so might impact the responsive resizing for JLab

**Comment by mrocklin** [comment]
Currently in the workers/ pane the `mem_plot` plot is very large.  We want it to take up only a small sliver of space.

Similarly we're getting strange effects when the user presses "reset" on the task stream plot in the status/ page.  The plot increases in vertical size.

**Comment by canavandl** [comment]
I think you should be able to constrain the mem_plot size by passing ``plot_height=300`` here: https://github.com/dask/distributed/blob/master/distributed/bokeh/workers/main.py#L21

I'll look into the task stream plot.

**Comment by canavandl** [comment]
It appears that ResetTool accepts a `reset_size` kwarg bool. While you described behavior sounds like a Bokeh bug, it may be resolved by adding ``reset_size=False`` to the ResetTool constructor in: https://github.com/dask/distributed/blob/master/distributed/bokeh/components.py#L91

**Comment by mrocklin** [comment]
Changes applied.  Lookin' sharp.

File: distributed/bokeh/components.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def __init__(self, n_rectangles=1000, **kwargs):
         self.root = Plot(
             title=Title(text="Task Stream"), id='bk-task-stream-plot',
             x_range=x_range, y_range=y_range, toolbar_location="above",
-            min_border_right=35, **kwargs
+            min_border_right=35, plot_height=300, **kwargs
```
Comment (canavandl) [inline distributed/bokeh/components.py]: also pass as kwarg here?

Comment (mrocklin) [inline distributed/bokeh/components.py]: I've removed it entirely.  This was only here as a hack to avoid the reset issue

**Comment by canavandl** [comment]
LGTM

**Pull request merged**